internal/flags.go

```diff
@@ -58,45 +64,34 @@ var (
 )
 
 type cfg struct {
-	Broker          string   `json:"broker"`
-	Username        string   `json:"username"`
-	Password        string   `json:"password"`
-	StorageLocation string   `json:"storageLocation"`
-	FeatureID       string   `json:"featureId"`
-	ModuleType      string   `json:"moduleType"`
-	ArtifactType    string   `json:"artifactType"`
-	Install         []string `json:"install"`
-	LogFile         string   `json:"logFile"`
-	LogLevel        string   `json:"logLevel"`
-	LogFileSize     int      `json:"logFileSize"`
-	LogFileCount    int      `json:"logFileCount"`
-	LogFileMaxAge   int      `json:"logFileMaxAge"`
+	Broker          string   `json:"broker" def:"tcp://localhost:1883" descr:"Local MQTT broker address"`
+	Username        string   `json:"username" descr:"Username for authorized local client"`
+	Password        string   `json:"password" descr:"Password for authorized local client"`
+	StorageLocation string   `json:"storageLocation" descr:"Location of the storage"`
+	FeatureID       string   `json:"featureId" def:"SoftwareUpdatable" descr:"Feature identifier of SoftwareUpdatable"`
+	ModuleType      string   `json:"moduleType" def:"software" descr:"Module type of SoftwareUpdatable"`
+	ArtifactType    string   `json:"artifactType" def:"archive" descr:"Defines the module artifact type: archive or plane"`
+	Install         []string `json:"install" descr:"Defines the absolute path to install script"`
+	ServerCert      string   `json:"serverCert" descr:"A PEM encoded certificate \"file\" for secure artifact download"`
+	LogFile         string   `json:"logFile" def:"logs/log.txt" descr:"Log file location in storage directory"`
+	LogLevel        string   `json:"logLevel" def:"INFO" descr:"Log levels are ERROR, WARNING, INFO, DEBUG, TRACE"`
```

Comment:
Could you please double check the warning log level. According to the documentation, it's: WARN.